readme.md

```diff
@@ -3,6 +3,7 @@
 
 ## Table of Contents
 
+- [Table of Contents](#table-of-contents)
```

Comment:
Please remove this added line

Reply:
I tried to remove it, but it was somehow added by itself. I will try to remove it one more time.